docs/components.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Components
+layout: default
+nav_order: 3
+---
+
+Operator-Lifecycle-Manager v0  was built as a [mono repo](https://github.com/operator-framework/operator-lifecycle-manager), which lead to an increase in the complexity of maintaining the project as it grew over time. 
```

Comment:
Nit: "Operator-Lifecycle-Manager" should be "Operator Lifecycle Manager". In other places throughout the docs when fully typing the name it appears as "Operator Lifecycle Manager" without the hyphens. The current v0 docs site also refers to it as "Operator Lifecycle Manager". For consistency we should remove the hyphenation